### ISSUE TITLE
unstable version. build error(visual studio 2019)

### ISSUE THREAD

**Issue by vhok74** [SoftEtherVPN/SoftEtherVPN#1450]
Hi. I'm building an unstable version in visual studio 2019. My current computer environment is Windows and x64 architecture.

As shown in "BUILD_WINDOWS.md," we opened the cmakelist and proceeded with the build, but the error appears as shown in the attached photo below. Do you know why? Please help me..

![12333](https://user-images.githubusercontent.com/35132299/128811350-01b2e161-22dd-465d-8968-3edae24cf188.PNG)

**Comment by davidebeatrici** [comment]
Could you confirm that #1451 fixes the issue, please?

**Comment by vhok74** [comment]
![image](https://user-images.githubusercontent.com/35132299/128954911-4a1a9a03-9255-4b32-9b35-ba578d3af59e.png)
I added the code to src/Cedar/CMakeList.txt as per the contents, but an error also occurs as shown in the picture above.

**Comment by davidebeatrici** [comment]
Sorry, the code was changed after my comment.

Add `PROPERTY` before `COMPILE_DEFINITIONS`.

**Comment by YangShizhao** [comment]
I also have confusion on building with VS2019. I found that the official document suggest that I should use VS2008 to build the project, and I have successully done it. But I have a new module which is based on VS2019, and after I build it with VS2019 and copy the static library to SoftEther, the build was failed, and I found that it is because different version of VS have different definition of functions.
So my question is: when can SoftEther support VS2019 officially?
